src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/PropertyPageFromProjectDataProducer.cs

```diff
@@ -1,13 +1,13 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+using Microsoft.VisualStudio.OLE.Interop;
```

Comment:
This is another questionable namespace. If this is actually used, then some type name in the file is no longer binding to the same type.

Reply:
Yep, not used. These were all incorrect comment imports.